### PR TITLE
Build Addons for Windows (CPU)

### LINKED ISSUE #173: Build Addons for Windows
Splitting this as a separate issue from #77. MacOS should be an achievable goal prior to the 0.3 release. Windows will take a bit longer.

### PR DESCRIPTION
**Pull request by seanpmorgan**
Fixes #173 

- [x] Compile custom ops on MSVC2017
- [x] Fix packaging of whl
- [x] Travis build 

TODO (Future Issues):
- [ ] Travis Testing (Win/MacOS)
- [ ] Parsetime needs to link a DLL with ABSL symbols exported (currently links with .pyd)
- [ ] Windows build in documentation (waiting for 0.7 release)
- [ ] Build GPU custom ops for windows

### CODE REVIEW

**Comment by seanpmorgan** [comment]
@tensorflow/sig-addons-maintainers ready for review. 

Built with travis and pushed to pypi test:
https://travis-ci.org/seanpmorgan/addons/builds/626967246
https://test.pypi.org/project/tensorflow-addons/0.7.0.dev0/#files

No py27 package since theres no binary for TF2.1.0rc1 but it should be there for the full release.

Tested on windows VM:
```
>>> import tensorflow as tf
>>> import tensorflow_addons as tfa
>>>
>>> print(tf.__version__)
2.1.0-rc1
>>> print(tfa.__version__)
0.7.0-dev
>>> tf.debugging.set_log_device_placement(True)
>>>
>>> x = tf.constant([[0.5, 1.2, -0.3]])
2019-12-18 19:29:22.789977: W tensorflow/stream_executor/platform/default/dso_loader.cc:55] Could not load dynamic library 'nvcuda.dll'; dlerror: nvcuda.dll not found
>>> layer = tfa.layers.GeLU()
>>> print(layer(x))
2019-12-18 19:29:35.200914: I tensorflow/core/common_runtime/eager/execute.cc:573] Executing op __inference_gelu_7 in device /job:localhost/replica:0/task:0/device:CPU:0
x: (_Arg): /job:localhost/replica:0/task:0/device:CPU:0
2019-12-18 19:29:35.209845: I tensorflow/core/common_runtime/placer.cc:54] x: (_Arg): /job:localhost/replica:0/task:0/device:CPU:0
Addons>Gelu: (Addons>Gelu): /job:localhost/replica:0/task:0/device:CPU:0
2019-12-18 19:29:35.217653: I tensorflow/core/common_runtime/placer.cc:54] Addons>Gelu: (Addons>Gelu): /job:localhost/replica:0/task:0/device:CPU:0
Identity: (Identity): /job:localhost/replica:0/task:0/device:CPU:0
2019-12-18 19:29:35.224838: I tensorflow/core/common_runtime/placer.cc:54] Identity: (Identity): /job:localhost/replica:0/task:0/device:CPU:0
identity_RetVal: (_Retval): /job:localhost/replica:0/task:0/device:CPU:0
2019-12-18 19:29:35.235247: I tensorflow/core/common_runtime/placer.cc:54] identity_RetVal: (_Retval): /job:localhost/replica:0/task:0/device:CPU:0
tf.Tensor([[ 0.345714    1.0617028  -0.11462908]], shape=(1, 3), dtype=float32)
```

File: README.md [hunk 1]
```diff
@@ -81,7 +81,7 @@ export CUDNN_INSTALL_PATH="/path/to/cudnn" (default: /usr/lib/x86_64-linux-gnu)
 # This script links project with TensorFlow dependency
 ./configure.sh
 
-bazel build build_pip_pkg
+bazel build --enable_runfiles build_pip_pkg
```
Comment (WindQAQ) [inline README.md]: NIT: remove trailing space

File: WORKSPACE [hunk 2]
```diff
@@ -1,5 +1,3 @@
-workspace(name = "tensorflow_addons")
-
```
Comment (WindQAQ) [inline WORKSPACE]: Why do we get rid of the workspace name here?

File: tensorflow_addons/custom_ops/activations/cc/kernels/rrelu_op.h [hunk 3]
```diff
@@ -38,7 +38,7 @@ struct Rrelu {
                   typename TTypes<T>::Tensor alpha,
                   typename random::SimplePhilox& random) {
     if (training) {
-      T storage[alpha.size()];
+      T* storage = new T[alpha.size()];
```
Comment (WindQAQ) [inline tensorflow_addons/custom_ops/activations/cc/kernels/rrelu_op.h]: cc @fsx950223

File: tools/ci_build/builds/release_windows.sh [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+set -e -x
+
+mkdir -p artifacts/
+export BAZEL_VC=/c/Program\ Files\ \(x86\)/Microsoft\ Visual\ Studio/2017/BuildTools/VC/
+
+# Install Bazel 0.24
```
Comment (WindQAQ) [inline tools/ci_build/builds/release_windows.sh]: NIT: 1.1.0

File: README.md (hunk 1 above)
Comment (seanpmorgan) [reply README.md]: Thanks!

File: tools/ci_build/builds/release_windows.sh (hunk 4 above)
Comment (seanpmorgan) [reply tools/ci_build/builds/release_windows.sh]: Thanks! Updated in macos_release as well

File: WORKSPACE (hunk 2 above)
Comment (seanpmorgan) [reply WORKSPACE]: Yeah, so I tried for a while to get this to work. Windows seemed unwilling to copy the files during `build_pip_pkg` if the directory was specified using workspace. Possibly related to windows sym-links but I'm not really sure. It's worth figuring out, but I've had about as much of windows build as I can stand for a while :P

The workspace name is something I added during CUDA10.1 PR and isn't being done for custom-op:
https://github.com/tensorflow/custom-op/blob/master/WORKSPACE

**Pull request merged**